src/screenshot.c

```diff
@@ -20,6 +20,7 @@
 
 #include "config.h"
 
+#include <stdint.h>
```

Comment:
```suggestion
```

you can remove this too now.

Reply:
Ah right I'm currently doing this on my phone  whoops 😬